app/code/Magento/Quote/Model/GuestCart/GuestShippingMethodManagement.php

```diff
@@ -11,7 +11,6 @@
 use Magento\Quote\Api\GuestShipmentEstimationInterface;
 use Magento\Quote\Api\ShipmentEstimationInterface;
 use Magento\Quote\Api\ShippingMethodManagementInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `getList`, `set`, `estimateByAddress` and `estimateByExtendedAddress` methods without the fully qualified name